server/controllers/finance/reports/journal/index.js

```diff
@@ -58,8 +66,10 @@ function postingJournalExport(req, res, next) {
       return report.render({ rows, totals });
     })
     .then((result) => {
+
       if (result.headers.type === 'xlsx') {
-        res.xls(result.headers.filename, util.dateFormatter(result.report.rows));
+        const rows = (renameKeys) ? util.renameKeys(result.report.rows, displayNames) : result.report.rows;
```

Comment:
It would be cool if we could do this for the CSV and XLS reporters.  Could we make this part of the [ReportManager.js](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/server/lib/ReportManager.js#L64) class?